File: app/src/main/java/com/ashu/callapitestcode/data/Api/APIDataUri.java
package com.ashu.callapitestcode.data.Api;

public interface APIDataUri {

    public static String getAQIUrl(String lat, String lon) {
        return "getNearestLocationV2?lat="+ lat +"&long="+ lon +"&type=1";
    }

    public static String getWeatherUrl(String locationid, String searchtype, String type) {
        return "getWeatherDetailsWithForecastApp?locationid="+ locationid +"&searchtype="+ searchtype +"&type=" + type;
    }

}

File: app/src/main/java/com/ashu/callapitestcode/data/Api/APIDataUri.kt
package com.ashu.callapitestcode.data.Api

object APIDataUri {

    fun getAQIUrl(lat: String, lon: String): String {
        return "getNearestLocationV2?lat=$lat&long=$lon&type=1"
    }

    fun getWeatherUrl(locationId: String, searchType: String, type: String): String {
        return "getWeatherDetailsWithForecastApp?locationid=$locationId&searchtype=$searchType&type=$type"
    }
}